### PR TITLE
plugins(prometheus) add new metrics since 0.9.x

### PR DESCRIPTION
**Pull request by fffonion**
### Review

@reviewer

### CODE REVIEW

**Review by kikito — APPROVED** [review]

File: app/_hub/kong-inc/prometheus/0.9.x.md [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+endpoint. Note that the URL to those APIs will be specific to your
+installation; see [Accessing the metrics](#accessing-the-metrics) below.
+
+This plugin records and exposes metrics at the node level. Your Prometheus
+server will need to discover all Kong nodes via a service discovery mechanism,
+and consume data from each node's configured `/metrics` endpoint.
+
+### Grafana dashboard
+
+Metrics exported by the plugin can be graphed in Grafana using a drop in
```
Comment (javierguerragiraldez) [inline app/_hub/kong-inc/prometheus/0.9.x.md:61]:
```suggestion
Metrics exported by the plugin can be displayed by Grafana using a drop in
```

File: app/_hub/kong-inc/prometheus/0.9.x.md [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+
+---
+
+Metrics are available on both the Admin API and Status API at the
+`http://localhost:<port>/metrics`
+endpoint. Note that the URL to those APIs will be specific to your
+installation; see [Accessing the metrics](#accessing-the-metrics) below.
+
+This plugin records and exposes metrics at the node level. Your Prometheus
+server will need to discover all Kong nodes via a service discovery mechanism,
```
Comment (HCloward) [inline app/_hub/kong-inc/prometheus/0.9.x.md:56]:
```suggestion
server will need to discover all Kong nodes by a service discovery mechanism,
```

File: app/_hub/kong-inc/prometheus/index.md [hunk 3]
```diff
@@ -79,6 +79,10 @@ dashboard: [https://grafana.com/dashboards/7424](https://grafana.com/dashboards/
   writing, and number of accepted connections.
 - **Target Health**: The healthiness status (`healthchecks_off`, `healthy`, `unhealthy`, or `dns_error`) of Targets
   belonging to a given Upstream.
+- **Dataplane Status**: The last seen timestamp, config hash and config sync status for
```
Comment (HCloward) [inline app/_hub/kong-inc/prometheus/index.md:82]:
```suggestion
- **Dataplane Status**: The last seen timestamp, config hash, and config sync status for
```

File: app/_hub/kong-inc/prometheus/index.md [hunk 4]
```diff
@@ -79,6 +79,10 @@ dashboard: [https://grafana.com/dashboards/7424](https://grafana.com/dashboards/
...
+data plane nodes are exported on control plane.
```
Comment (HCloward) [inline app/_hub/kong-inc/prometheus/index.md:83]:
```suggestion
data plane nodes is exported to control plane.
```

**Review by HCloward — APPROVED** [review]
Some minor wording suggestions, otherwise LGTM! Thanks @fffonion !

**Pull request merged**